roles/cluster-inventory/tasks/main.yml

```diff
@@ -0,0 +1,45 @@
+---
+- name: Get floating IP of nodes in a Magnum cluster using Openstack CLI
+  shell: "openstack coe cluster show {{ cluster_name }} -c master_addresses -c node_addresses -f json"
+  register: result_addresses
+
+- name: Get a list of all running servers using Openstack CLI
+  shell: "openstack server list -c ID -c Networks -f json --user {{ cluster_owner | default(ansible_user) }}"
```

Comment:
In general, it's better to use the command module as opposed to shell to isolate yourself from the user's env.

Reply:
Although in this case we need the user's environment, right?

Reply:
That's true - so the plan is to get the user to source their credentials before running this?